web3_proxy/tests/common/get_rpc_key.rs

```diff
@@ -0,0 +1,58 @@
+use crate::TestApp;
+use ethers::prelude::{LocalWallet, Signer};
+use ethers::types::Signature;
+use reqwest::Response;
+use rust_decimal::Decimal;
+use serde::Deserialize;
+use tracing::info;
+use web3_proxy::frontend::admin::AdminIncreaseBalancePost;
+use web3_proxy::frontend::users::authentication::{LoginPostResponse, PostLogin};
+
+#[derive(Debug, Deserialize)]
+pub struct RpcKeyResponse {
+    pub user_id: u64,
+    pub user_rpc_keys: std::collections::HashMap<String, RpcKey>,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct RpcKey {
+    pub active: bool,
+    pub allowed_ips: Option<serde_json::Value>,
+    pub allowed_origins: Option<serde_json::Value>,
+    pub allowed_referers: Option<serde_json::Value>,
+    pub allowed_user_agents: Option<serde_json::Value>,
+    pub description: Option<serde_json::Value>,
+    pub id: u64,
+    pub log_revert_chance: f64,
+    pub private_txs: bool,
+    pub role: String,
+    pub secret_key: String,
```

Comment:
should we have a stricter type on this? (ulid or RpcSecretKey)

Reply:
I thought about keeping it String just because that's what the client will interpret it as. can change it to Ulid though.

Reply:
Lets do Ulid. There was one time I messed up and was returning them as Uuid by accident. This test could catch that